css/styles.css

```diff
@@ -0,0 +1,48 @@
+html,
+body {
+    margin: 0;
```

Comment:
It would be better to add this margin and padding resetting to universal selector.
Specify basic font and color details to body tag.